test/gtest/ucs/test_time.cc

```diff
@@ -103,8 +102,7 @@ UCS_TEST_F(test_time, timerq) {
          */
         counter1 = 0;
         counter2 = 0;
-        status = ucs_timerq_remove(&timerq, TIMER_ID_1);
-        ASSERT_UCS_OK(status);
+        ASSERT_UCS_OK(ucs_timerq_remove(&timerq, TIMER_ID_1));
```

Comment:
why removed "status" variable?

Reply:
"status" var can be changed to "ASSERT_UCS_OK", seemed cleaner.
I'll change it back if you prefer.